dipy/denoise/tests/test_nlmeans.py

```diff
@@ -18,14 +18,14 @@ def test_nlmeans_padding():
 
 def test_nlmeans_static():
     S0 = 100 * np.ones((20, 20, 20), dtype='f8')
-    S0n = nlmeans(S0, sigma=1, rician=False)
+    S0n = nlmeans(S0, sigma=np.ones((20, 20, 20)), rician=False)
```

Comment:
I wouldn't replace the test here. Instead add the new test, together with the old one.